jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/skaffold/FilesTaskV2.java

```diff
@@ -176,7 +177,10 @@ private void addProjectFiles(Project project) {
 
       // Search through all dependencies
       for (Configuration configuration :
-          currentProject.getConfigurations().getByName("runtime").getHierarchy()) {
+          currentProject
+              .getConfigurations()
+              .getByName(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME)
```

Comment:
@lightoze in this class, there's another use of `getByName("runtime")` for SNAPSHOT, non-project dependencies. Does it need to be changed as well?

```java
    for (File file : project.getConfigurations().getByName("runtime")) {
      if (!projectDependencyJars.contains(file) && file.toString().contains("SNAPSHOT")) {
```

And you said this is broken with Gradle 5. After changing this, will it continue to work with Gradle 4.9? (Gradle 4.9 is the minimum version Jib supports/requires.)

@loosebazooka looks like the value is now changed to "runtimeClasspath" from "runtime." Does this make sense?

Reply:
@chanseokoh probably that should be changed too, but it does not affect the project I have on hands so I have nothing to test on. As such "runtime" is deprecated and should be used if only for compatibility.
Gradle 4 should continue to work but I haven't tested.

Reply:
Yeah I think with 2.0 we plan on going to minimum of gradle 5 anyway: https://github.com/GoogleContainerTools/jib/issues/2140

Reply:
Also, yeah I agree we should use `runtimeClasspath` everywhere...

Reply:
@chanseokoh @loosebazooka OK changed that too and reworked patch into a single commit.